packages/payment-integration-api/src/errors/custom-error-types.ts

```diff
@@ -0,0 +1,13 @@
+export interface SpecificError extends Error {
+    errors?: ErrorElement[];
+    status?: string;
+}
+
+interface ErrorElement {
+    code: string;
+    message: string;
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    provider_error?: {
```

Comment:
Can we map this key to something that follows the naming convention so we don't need to disable rules?

Reply:
it is come from back-end like this